src/Engine/ProtoCore/Reflection/GraphicDataProvider.cs

```diff
@@ -147,21 +147,24 @@ internal List<IGraphicItem> GetGraphicItems(DSASM.StackValue svData, RuntimeCore
         
         internal object GetCLRObject(StackValue svData, RuntimeCore runtimeCore)
         {
-            if (marshaler != null)
-            {
-                return marshaler.UnMarshal(svData, null, interpreter, typeof(object));
-            }
-            
             if (null == runtimeCore.DSExecutable.classTable)
                 return null;
 
-            IList<ClassNode> classNodes = runtimeCore.DSExecutable.classTable.ClassNodes;
-            if (null == classNodes || (classNodes.Count <= 0))
+            //The GetCLRObject function is typically utilized to retrieve a ClrObject from a StackValue of type pointer.
+            //There is an edge cases for pointers where the pointer references a non CLR object.  This code
+            //checks for this edge case by verifying that the requested StackValue pointer is associated with an
+            //imported library.  An example is the "Function" pointer which does not have an associated CLRObject.
+            //In that case, the return value should be null.
+            var classNode = runtimeCore.DSExecutable.classTable.GetClassNodeAtIndex(svData.metaData.type);
+            if (classNode != null  && !classNode.IsImportedClass)
```

Comment:
Is checking if it's not an imported class a sufficient condition to determine that it's a function object? My concern is that there could be (if not today) in the future, other DS types that are not treated as imported classes for which we may not want to return null.

Reply:
Can we use this condition instead?

Reply:
Based on our discussion, I think it's okay to leave this as-is for now. LGTM!

Reply:
This may have been a pre-existing issue then.  This fix just returns the function to its previous behavior.

Reply:
This may have been a pre-existing issue then.  This fix just returns the function to its previous behavior.